src/module.c

```diff
@@ -2254,6 +2254,33 @@ int moduleIsModuleCommand(void *module_handle, struct serverCommand *cmd) {
     return (cp->module == module_handle);
 }
 
+/* ValkeyModule_UpdateRuntimeArgs can be used to update the values of module->loadmod
+ * so that the updated values can be saved into conf file once 'config rewrite' command
+ * is called
+ * One example can be found in file modules/moduleparameter.c
+ *
+ * Returns:
+ * - VALKEYMODULE_OK on successfully updating.
+ * - VALKEYMODULE_ERR on failure.
+ */
+int VM_UpdateRuntimeArgs(ValkeyModuleCtx *ctx, int argc, ValkeyModuleString **argv) {
```

Comment:
All the other API functions that have argv/argc have argv before argc. We should do the same here.

For example OnLoad:

    int ValkeyModule_OnLoad(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc);

Reply:
Addressed, I just found the  order of parameters argv and argc is reversed to C main function

Reply:
> Addressed, I just found the order of parameters argv and argc is reversed to C main function

Yes, :joy: probably it was an early mistake when the module API was created. But now it's better to follow this style.